server/seed/index.js

```diff
@@ -0,0 +1,51 @@
+require("dotenv").config();
+const colors = require("colors");
+const User = require("../models/User");
+const Profile = require("../models/Profile");
+const { userArray, profileArray } = require("./sampleProfiles");
+const connectDB = require("../db");
+
+connectDB();
+
+console.log("loading mock users");
+userArray.forEach(async (user) => await User.create(user));
```

Comment:
You can use a `for` loop to handle this because it's weird how JavaScript `forEach` method handles `async` callbacks.